src/test/resources/application.properties

```diff
@@ -1,21 +1,11 @@
-spring.application.name=car-sharing-service
-
-spring.datasource.url=jdbc:h2:mem:testdb
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=
-spring.datasource.password=
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.liquibase.enabled=true
-spring.jpa.hibernate.ddl-auto=validate
-spring.jpa.show-sql=true
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
-server.servlet.context-path=/api
-bot.token=${TELEGRAM_BOT_TOKEN}
-bot.name=${TELEGRAM_BOT_USERNAME}
-stripe.api.key=${STRIPE_API_KEY}
+spring.datasource.url=jdbc:tc:mysql:8:///TEST_DB_URL
+spring.datasource.username=TEST_DB_USER
+spring.datasource.password=TEST_DB_PASSWORD
 
 jwt.secret=${JWT_SECRET}
 jwt.expiration=${JWT_EXPIRATION}
 
 openapi.module-name=Car Sharing Service
 openapi.api-version=1.0
+spring.profiles.active=test
+stripe.api.key=1
```

Comment:
Good, but i think we need here liquibase too, at least scripts for creating tables